mininet/node.py

```diff
@@ -838,6 +838,38 @@ def init( cls ):
         mountCgroups()
         cls.inited = True
 
+class XIAHost( Host ):
+
+    "XIA host"
+
+    def __init__( self, name, **kwargs ):
+        ""
+       	Node.__init__( self, name, **kwargs )
+	self.xids = {}        
```

Comment:
This line, and many of the lines below, uses a tab for an indent. However, this file uses spaces for indentation. To adhere to the style being used, remove all of the tabs being used for indentation in the code you've added here.